pkg/networkservice/common/excludedprefixes/client.go

```diff
@@ -190,6 +190,10 @@ func (epc *excludedPrefixesClient) Close(ctx context.Context, conn *networkservi
 func getRoutePrefixes(routes []*networkservice.Route) []string {
 	var rv []string
 	for _, route := range routes {
+		var o, b = route.GetPrefixIPNet().Mask.Size()
+		if o != b {
+			continue
+		}
```

Comment:
This seems to only compare the mask sizes between routes... I don't quite follow how this precludes us from including a Connection's routes/srcAddr/dstAddr in the exclude_prefixes for Refresh requests.

Reply:
as I can see srcAddr/dstAddr is already supported we only without a reason exclude a whole subnet. This patch prevents excluding a subnet.

Reply:
That will work for *now*.  But we need to be more specific in in the medium term because the routes from *other* vWires should be included :)